packages/commands/src/zetachain/call.ts

```diff
@@ -73,6 +82,7 @@ ZetaChain Gateway: ${gatewayZetaChain}
 Function: ${options.function}
 Function parameters: ${options.values?.join(", ")}
 Parameter types: ${stringifiedTypes}
+Withdraw Gas Fee: ${gasFee} ${gasSymbol}
```

Comment:
Does this make sense here in `call`?

Reply:
When making a non-asset call from ZetaChain you're not withdrawing anything, but you're making a contract call on a connected chain, so you're paying a fee to make this call. The fee amount is determined by the amount of gas you expect this call will consume on a connected chain.

Reply:
It's a bit counterintuitive on the protocol side that this is called withdraw fee, both for actual withdrawals as well as for no-asset calls.